src/Symfony/Component/DependencyInjection/Compiler/ResolveBindingsPass.php

```diff
@@ -185,6 +186,9 @@ protected function processValue(mixed $value, bool $isRoot = false): mixed
                 if (\array_key_exists($parameter->name, $arguments) && '' !== $arguments[$parameter->name]) {
                     continue;
                 }
+                if ($parameter->getAttributes(Autowire::class, \ReflectionAttribute::IS_INSTANCEOF)) {
```

Comment:
we should at least check the `'container.ignore_attributes'` tag here
and also if the class is autowired

Reply:
Done